scripts/joint_trajectory_file_playback.py

```diff
@@ -233,12 +239,38 @@ def find_start_offset(pos):
             cur_cmd = [cmd['right_gripper']]
             self._add_point(cur_cmd, 'right_gripper', values[0] + start_offset)
 
+    def _feedback(self, data):
+        if not self._get_trajectory_flag():
+            self._set_trajectory_flag(value=True)
+
+    def _set_trajectory_flag(self, value=False):
+        # Lock
+        self._lock.acquire()
+        # Assign a value to the flag
+        self._arm_trajectory_started = value
+        # Unlock
+        self._lock.release()
+
+    def _get_trajectory_flag(self):
+        temp_flag = False
+        # Lock
+        self._lock.acquire()
+        # Deep copy to external variable
+        temp_flag = copy.deepcopy(self._arm_trajectory_started)
```

Comment:
I wonder if you need the temp_flag too ... I don't know when resolution / copy occurs with immutables